aspnetcore/signalr/javascript-client.md

```diff
@@ -263,7 +263,47 @@ The following code demonstrates a typical manual reconnection approach:
 
 [!code-javascript[](javascript-client/samples/3.x/SignalRChat/wwwroot/chat.js?range=30-42)]
 
-A real-world implementation would use an exponential back-off or retry a specified number of times before giving up.
+A production implementation would use an exponential back-off or retry a specified number of times before giving up.
+
+<!-- This heading is used by code in the SignalR Typescript client, do not rename or remove without considering the impacts there -->
+
+<h2 id="bsleep">Browser sleeping tab</h2>
```

Comment:
@richlander @guardrex Can you give this a quick final once over and then I'll merge

Reply:
I defer to @Rick-Anderson on this. I only use this when I must; and I think with *version by file* now :tada:, it will only be needed in Blazor docs that use a WASM/Server pivot with duplicate headings.